caluma/caluma_form/signals.py

```diff
@@ -93,67 +85,3 @@ def remove_calc_dependents(sender, instance, **kwargs):
     update_calc_dependents(
         instance.slug, old_expr=instance.calc_expression, new_expr="false"
     )
-
-
-# Update calculated answer on post_save
-#
-# Try to update the calculated answer value whenever a mutation on a possibly
-# related model is performed.
-
-
-@receiver(post_save, sender=models.Question)
-@disable_raw
-@filter_events(lambda instance: instance.type == models.Question.TYPE_CALCULATED_FLOAT)
-def update_calc_from_question(sender, instance, created, update_fields, **kwargs):
-    # TODO optimize: only update answers if calc_expression is updated
-    # needs to happen during save() or __init__()
-
-    for document in models.Document.objects.filter(form__questions=instance):
-        update_or_create_calc_answer(instance, document)
-
-
-@receiver(post_save, sender=models.FormQuestion)
-@disable_raw
-@filter_events(
-    lambda instance: instance.question.type == models.Question.TYPE_CALCULATED_FLOAT
-)
-def update_calc_from_form_question(sender, instance, created, **kwargs):
-    for document in instance.form.documents.all():
-        update_or_create_calc_answer(instance.question, document)
```

Comment:
I dropped these two handlers completely. On my local instance, updating a calculated question took about 10s, even though I only had 10 documents in the DB. Since in typical prod scenarios we have thousands of documents, this kind of thing should be done, if needed, explicitly in a management command.